### ISSUE TITLE
[Bug]: crash on station guard mode change on latest beta

### ISSUE THREAD

**Issue by thieren** [homebridge-plugins/homebridge-eufy-security#121]
### What happened?

On changing the guard mode on latest beta with eufy-security-client v.2.1.1 the plugin crashes.

### Device Type

Doorbell (Supported)

### Plugin Version

beta (Best Effort)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v16 (Supported)

### Operating System

Windows 10+ (Supported)

### Relevant log output

```shell
[7/21/2022, 8:51:34 AM] [EufySecurity-2.1.0-beta.11] DEBUG: Homebase SET StationGuardMode (raw value homekit):0
[7/21/2022, 8:51:34 AM] [EufySecurity-2.1.0-beta.11] DEBUG: Homebase SET StationGuardMode:1
[7/21/2022, 8:51:34 AM] [EufySecurity-2.1.0-beta.11] INFO:  Homebase Request to change station guard mode to: Home.
[7/21/2022, 8:51:34 AM] [EufySecurity-2.1.0-beta.11] DEBUG: Homebase ON handlePropertyChange: { name: 'guardMode', value: 1 }
[7/21/2022, 8:51:34 AM] [EufySecurity-2.1.0-beta.11] DEBUG: Homebase ON handleRawPropertyChange: { type: 1151, value: '1' }
[7/21/2022, 8:51:34 AM] [EufySecurity-2.1.0-beta.11] DEBUG: Homebase ON SecuritySystemCurrentState: 1
[7/21/2022, 8:51:34 AM] [EufySecurity-2.1.0-beta.11] DEBUG: Homebase ON handlePropertyChange: { name: 'currentMode', value: 1 }
TypeError: Cannot read properties of undefined (reading 'toString')
    at Station.getArmDelay (C:\Users\Rene\AppData\Roaming\npm\node_modules\homebridge-eufy-security\node_modules\eufy-security-client\src\http\station.ts:596:85)
    at Station.onAlarmMode (C:\Users\Rene\AppData\Roaming\npm\node_modules\homebridge-eufy-security\node_modules\eufy-security-client\src\http\station.ts:566:31)
    at P2PClientProtocol.<anonymous> (C:\Users\Rene\AppData\Roaming\npm\node_modules\homebridge-eufy-security\node_modules\eufy-security-client\src\http\station.ts:54:68)
    at P2PClientProtocol.emit (node:events:527:28)
    at P2PClientProtocol.handleDataControl (C:\Users\Rene\AppData\Roaming\npm\node_modules\homebridge-eufy-security\node_modules\eufy-security-client\src\p2p\session.ts:1274:22)
    at P2PClientProtocol.handleData (C:\Users\Rene\AppData\Roaming\npm\node_modules\homebridge-eufy-security\node_modules\eufy-security-client\src\p2p\session.ts:988:18)
    at P2PClientProtocol.parseDataMessage (C:\Users\Rene\AppData\Roaming\npm\node_modules\homebridge-eufy-security\node_modules\eufy-security-client\src\p2p\session.ts:979:26)
    at P2PClientProtocol.handleMsg (C:\Users\Rene\AppData\Roaming\npm\node_modules\homebridge-eufy-security\node_modules\eufy-security-client\src\p2p\session.ts:801:26)
    at Socket.<anonymous> (C:\Users\Rene\AppData\Roaming\npm\node_modules\homebridge-eufy-security\node_modules\eufy-security-client\src\p2p\session.ts:113:56)
    at Socket.emit (node:events:527:28)
[21.7.2022, 08:51:34] [homebridge-eufy-security] Child bridge process ended
```
```

**Comment by thieren** [comment]
fixed in 2.1.0-beta.14

**Comment by thieren** [comment]
Is fixed with release of 2.1.0
